app/models/dns-record.server.ts

```diff
@@ -126,3 +126,13 @@ export function getExpiredDnsRecords() {
     },
   });
 }
+
+/**
+ * This Fn gets the base data that is needed for reconciliation
+ * but does that for all records in the db. Should be no more than 10k
+ */
```

Comment:
It's worth adding a note here that this is our ground truth for all details about DNS Records.